docs/guide/getting-started.md

```diff
@@ -233,4 +234,55 @@ We've already covered how to build Superstreamer locally, and we've also made it
 $ bun run dev
 ```
 
-:::
\ No newline at end of file
+### Quick Development Environment Setup
+
+We have also created a `docker-compose-dev.yml` so you can setup your development environment faster and start getting hands on!
+
+```shell
+# We have prebuilt developmnet containers, see docker/docker-compose-dev.yml
+cd docker
+docker-compose -f docker-compose-dev.yml up
+```
+
+You can use this `config.env.development` for a quick setup:
```

Comment:
Since `config.env.development` as file name would be ignored, it's probably best to mention that people have to paste these env variables in `config.env` instead.

Reply:
I added a clarification:

> "You can create a file named `config.env.development` for a quick setup. Here is a sample that should work out of the box if default configuration is used:"